packages/editor-ui/src/components/Node.vue

```diff
@@ -106,12 +113,37 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 
 			return workflowResultDataNode.length;
 		},
+		canvasOffsetPosition() {
+			return this.$store.getters.getNodeViewOffsetPosition;
+		},
 		isExecuting (): boolean {
 			return this.$store.getters.executingNode === this.data.name;
 		},
+		isDragActive (): boolean {
+			return this.$store.getters.isActionActive('dragActive');
+		},
+		isSingleActiveTriggerNode (): boolean {
+			const nodes = this.$store.getters.activeWorkflowTriggerNodes.filter((node: INodeUi) => {
+				return this.$store.getters.nodeType(node.type).eventTriggerDescription !== '' && !node.disabled;
```

Comment:
nodeType can return null. always check return type of getters. 
```suggestion
        const nodeType =  this.$store.getters.nodeType(node.type) as INodeUi | null;
				return nodeType && nodeType.eventTriggerDescription !== '' && !node.disabled;
```

Reply:
Done

Reply:
@mutdmour I updated once again. I think this should be 
const nodeType =  this.$store.getters.nodeType(node.type) as INodeTypeDescription | null;

(INodeTypeDescription instead of  INodeUI